haskell-overlays/text-jsstring/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, haskellLib, fetchFromGitHub, fetchpatch, versionWildcard, thunkSet }:
+
+with lib;
+with haskellLib;
+
+self: super: {
+  _dep = super._dep or {} // thunkSet ./dep // {
+  #   ghcjsBaseTextJSStringSrc = super.ghcjs-base.src.overrideAttrs (drv: {
```

Comment:
Why's this stuff commented out?

Reply:
Oh good catch, this is just the deduplicated parts. I will clean up.

Reply:
No more comments here.